src/site/pages/upgrade-guide.md

```diff
@@ -0,0 +1,223 @@
+# PatternFly 4 Upgrade Guide
+
+Upgrading your environment from PatternFly 3 to PatternFly 4.
+
+**Important**
+
+When upgrading from a PatternFly 3 environment to a PatternFly 4 environment, you will need to do the following:
+
+- Verify that your application does not need IE 11 support. If it does, you cannot upgrade to PatternFly 4.
+- Check your build process. The steps for basic HTML, Gulp and Webpack differ and have different requirements.
+- Determine if you need to run PatternFly 3 and PatternFly 4 together. If you do, you will need to update the following configuration in the `src/patternfly/utilities/variables.scss` file to state `true`.
+
+```scss
+$pf-global--load-pf-3: true !default;
+```
+
+
+## Installing PatternFly 4
+
+```json
+npm install @patternfly/patternfly-next
+```
+
+----
+
+## Configuration
+
+Inside of the patternfly-next package, you will want to find and copy `patternfly.css` to wherever you keep your site CSS files. After doing so, add the following line last in your list of CSS files:
+
+```html
+<link rel="stylesheet" href="css/patternfly.css">
+```
+
+This will make it so that PatternFly 4 styles take precedence over anything that currently resides in your application.
+
+## Building PatternFly 4
+
+PatternFly 4 is distributed as separated modules, so that you can consume pieces on a layout and component basis. Each module delivers a sass file (`scss`) and `css` file, so that you can either include them in your build environment, or just consume the CSS in your page header.
```

Comment:
We might want to introduce the terms here - what's a module v component v layout. 
PatternFly 4 is distributed as separated modules:

- **Layouts** allow you to structure and organize the content on your pages 
- **Components**, like buttons and alerts, can be assembled together to build applications

Each module delivers a sass file (`scss`) and `css` file so you can either include them in your build environment or consume the CSS from your page header.